### PR TITLE
[ci skip] updated ruby doc links; for rails next version of ruby will be 2.2x

### PR DESCRIPTION
**Pull request by Gaurav2728**

### CODE REVIEW

**Comment by Gaurav2728** [comment]
@senny i replaced all major ruby-doc links with 2.2.2 version, rails 5.0 will be supporting that and it may be helpfull for reference.

File: guides/source/testing.md [hunk 1]
```diff
@@ -484,7 +484,7 @@ All the basic assertions such as `assert_equal` defined in `Minitest::Assertions
 
 Each of these classes include `Minitest::Assertions`, allowing us to use all of the basic assertions in our tests.
 
-NOTE: For more information on `Minitest`, refer to [Minitest](http://ruby-doc.org/stdlib-2.1.0/libdoc/minitest/rdoc/MiniTest.html)
+NOTE: For more information on `Minitest`, refer to [Minitest](http://ruby-doc.org/stdlib-2.2.2/libdoc/minitest/rdoc/MiniTest.html)
```
Comment (arunagw) [inline guides/source/testing.md]: The new links seems to be broken

Comment (senny) [inline guides/source/testing.md]: :sweat: ... Probably better to link to minitest outside of stdlib. We usually require a higher version anyway.

@arunagw do you want to update the link accordingly?

Comment (arunagw) [inline guides/source/testing.md]: @senny will do

**Comment by senny** [comment]
@Gaurav2728 please be conservative when submitting such PR's. As [this comment](https://github.com/rails/rails/pull/20222/files#r30695327) illustrates, the potential for failure is high and the end-user benefit is very low in comparison. As Rails 5 will be 2.2. only this update seems reasonable but please don't go through the docs and update all the version links.

Thank you for your understanding.

File: guides/source/testing.md (hunk 1 above)
Comment (senny) [inline guides/source/testing.md]: @arunagw thanks for catching this! :yellow_heart:

**Comment by Gaurav2728** [comment]
@senny @arunagw :sweat: thanks, i'll keep that in mind :+1:

**Pull request merged**